lxc/lxd
# Shared: Remove requirement on cgo when setting raw mode

Tests done:

1. Check ctrl+c works inside an interactive terminal (using `htop`).
2. Check backspace is OK once exited interactive terminal.
3. Check window resize works.
